python/doc/examples/functional_modeling/vectorial_functions/plot_functions_outputDim.py

```diff
@@ -0,0 +1,79 @@
+"""
+===========================================
+Increase the output dimension of a function
+===========================================
+"""
+# %%
+#
+# Import the required modules
+
+# %%
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a function :math:`f : \mathbb{R}^d \mapsto \mathbb{R}^q` from *q* functions :math:`f_i: \mathbb{R}^d \mapsto \mathbb{R}`.
+#
+# We can do that:
+#
+#     - Case 1: by aggregation  of the functions :math:`f_i`,
+#     - Case 2: by creating a vectorial linear combination  of the functions :math:`f_i`.
+
+
+# %%
```

Comment:
This produces a large blank space:

![image](https://user-images.githubusercontent.com/31351465/230657399-64690223-aa63-4d34-abc4-24388439b26d.png)

```suggestion
```

Reply:
ok